psydac/api/feec.py

```diff
@@ -49,9 +52,8 @@ class DiscreteDerham(BasicDiscrete):
     - For the multipatch counterpart of this class please see
       `MultipatchDiscreteDerham` in `psydac.feec.multipatch.api`.
     """
-    def __init__(self, mapping, domain_h, *spaces):
+    def __init__(self, domain_h, *spaces):
```

Comment:
this comment is for the lines above: the docstring seems obsolete: the class is now in the same file and called `DiscreteDerhamMultipatch`